components/BottomNavigation/README.md

```diff
@@ -181,3 +181,82 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+## Accessibility {#a11y}
+
+To help ensure your bottom navigation item is accessible to as many users as possible, please
+be sure to review the following recommendations:
+
+### `-accessibilityLabel`
+The label will be the title of the UITabBarItem. Currently you can't set this to a custom value.
+
+### Set `-accessibilityValue` 
+
+Set an appropriate
+[`accessibilityValue`](https://developer.apple.com/documentation/uikit/uiaccessibilityelement/1619583-accessibilityvalue)
```

Comment:
It doesn't look like we're using UITabBarItem.accessibilityValue for anything. Did you confirm that it works as expected?

Reply:
Yes.